src/bundle/@wdio-utils.ts

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const bundle = require("./cjs");
```

Comment:
Can we not use require here?
If import does not work here, i think `const bundle = import("./cjs")` would.